magni_bringup/scripts/launch_core.py

```diff
@@ -185,40 +223,23 @@ def main():
     )
     parser.add_argument(
         "--launch_generate_path",
-        default="/tmp/core.launch",
-        help="Generated the launch file to this path",
+        default=rp.get_path("magni_bringup")+"/launch/generated_core.launch",
+        help="Generate the launch file to this path",
     )
     arguments, unknown = parser.parse_known_args()
 
-    conf_path = find_file_by_priority(conf_path_1, conf_path_2)
-    if conf_path == "":
-        print(
-            "ERROR: configuration file robot.yaml could not be found in either:\n"
-            + conf_path_1
-            + " OR\n"
-            + conf_path_2
-        )
-        return
-    else:
-        print("Found config file: " + conf_path)
-
-    conf = get_yaml(conf_path, default_conf)
+    conf = get_config_replace_missing(conf_path, default_conf)
 
-    # We only support 1 version of the Sonars right now
-    if conf["sonars"] == "pi_sonar_v1":
-        conf["sonars"] = True
-    else:
-        conf["sonars"] = False
+    # print out the whole config if in debug mode
+    if arguments.debug == True:
+        print("DEUBG: Full content of the applied config:")
```

Comment:
Cleanup:    Is this really needed?  This may be a debug artifact

Reply:
No, "debug" is parameter you can run the launch_core.py with to get some usefull info out of it. Usefull in the future